docs/API.md

```diff
@@ -1458,7 +1458,7 @@ client.remove_object(
 
 ### remove_objects(bucket_name, delete_object_list, bypass_governance_mode=False)
 
-Remove multiple objects.
+Removes multiple objects. Uses the "DeleteObjects" S3 API call under the hood, which can delete up to 1000 keys with a single request. If `delete_object_list` contains more than 1000 entries, `remove_objects` splits it into smaller chunks and calls "DeleteObjects" multiple times.
```

Comment:
As `minio-py` works as per S3 specification, why do we need to document low level API behavior?

Reply:
Agreed, the behavior explanation is superfluous here. Our API handles all these nuances underneath.

Reply:
As an user of minio-py, I was about to pass 8000+ keys to to `remove_objects`. I knew from AWS documentation that the "DeleteObjects" API call accepts up to 1000 keys in one request. I was not sure how `remove_objects` would handle it – would it try to pass all 8000 keys, would it pass first 1000, would it make multiple requests with smaller batches, would it throw an exception, or do something else. To find out, I had to look at the code. A note in documentation like in this PR would have been helpful to me.